Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -71,7 +71,7 @@
     soundHit: /Audio/Weapons/Guns/Hits/snap.ogg
     damages:
       Blunt: 3
-  - type: StunnableProjectile
+  - type: StunOnCollide
     paralyzeAmount: 2
```

Comment:
no datafield called paralyzeAmount on the comp?

Reply:
paralyze is stun+knockback so this should be `stunAmount` and `knockbackAmount` now